python/tools/decoder.py

```diff
@@ -80,7 +80,7 @@
 for i in range(0, encoded_length, frame_length):
 
     lc3_frame_size = struct.unpack('=H', f_lc3.read(2))[0]
-    pcm = dec.decode(f_lc3.read(lc3_frame_size), bitdepth=bitdepth)
+    pcm = dec.decode(f_lc3.read(lc3_frame_size), bit_depth=bitdepth)
```

Comment:
I think the argument name can be changed to `bit_depth` too.

Reply:
Done